tox.ini

```diff
@@ -21,11 +22,15 @@ basepython =
     py3: python3.8
 
 usedevelop = true
-passenv = DISPLAY
+
+[testenv:functional3]
+passenv = DISPLAY WEBDRIVER DISPLAY FIREFOX_PATH TRAVIS URL WAITTOSTART CONTAINERTZ
```

Comment:
`DISPLAY` appears twice.

Reply:
> The instructions in this issue should be added to the contributing.md in Deform. I would also like to include a comment that refers to how to configure [docker-selenium](https://github.com/SeleniumHQ/docker-selenium).

@stevepiercy I have made some changes, and it is almost done. starting documenting it in contributing.md.
how do you verify your changes to contributing.md ? locally or after every push in github ?

Reply:
I took on revising contributing.md in https://github.com/Pylons/deform/pull/473. I think we're good.